src/lib/hooks/useClipboard.js

```diff
@@ -1,53 +1,78 @@
-import { useCallback } from 'react'
+import React, { useCallback } from 'react'
+import { Image, Platform } from 'react-native'
+
+import ExplanationDialog from '../../components/common/dialogs/ExplanationDialog'
+import illustration from '../../assets/ClipboardPermissionError.svg'
+
+import { useDialog } from '../undux/utils/dialog'
+
+import { isSafari } from '../utils/platform'
 import Clipboard from '../utils/Clipboard'
 import logger from '../logger/pino-logger'
 
 const log = logger.child({ from: 'useClipboard Hook' })
 
-const logPermissionsDenied = (exception, content = null) => {
-  const { name, message } = exception
+if (Platform.OS === 'web') {
+  Image.prefetch(illustration)
```

Comment:
we use svg component no? so i dont think we need this anymore

Reply:
No, we don't. There is no svgr implementation in the master branch. Only in the react-native branch. There is a task already created to move it from react-native to master.
https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/1933